### PR TITLE
Always use the language in URLs

### PR DESCRIPTION
**Pull request by zazabe**
- make the language always available on env, from the url, the user pref or a default one 
- use the language in all URLs
- support edge case, when there's no language defined in `cm_model_language` (in tests only) -> language abbreviation will not be added to URLs

### CODE REVIEW

**Comment by zazabe** [comment]
related to https://github.com/cargomedia/cm/issues/2486

**Comment by zazabe** [comment]
@njam should we redirect url without language, like `/foo` => `/en/foo`? seems not so easy, we don't know the page in https://github.com/zazabe/cm/blob/8b06bbda414b9768493d68617ae6bbc25c28329a/library/CM/Http/Response/Page.php#L70...

**Comment by njam** [comment]
We should not redict. The default-language version should still be accessible, be it only for Google.

**Comment by zazabe** [comment]
@njam please review

File: library/CM/RenderAdapter/Document.php [hunk 1]
```diff
@@ -99,9 +99,8 @@ private function _getOptions() {
         $options['urlCdn'] = $this->getRender()->getSite()->getUrlCdn();
         $options['urlUserContentList'] = $serviceManager->getUserContent()->getUrlList();
         $options['urlServiceWorker'] = $this->getRender()->getUrlServiceWorker();
-        if ($this->getRender()->getLanguageRewrite()) {
-            $options['urlLanguage'] = $this->getRender()->getLanguage();
-        }
+        $options['urlLanguage'] = $this->getRender()->getLanguage();
```
Comment (njam) [inline library/CM/RenderAdapter/Document.php]: Rename this to just `language`?

**Comment by njam** [comment]
lgtm
